src/v/redpanda/admin/security.cc

```diff
@@ -103,6 +106,18 @@ bool is_no_op_user_write(
     }
 }
 
+bool is_no_op_role_write(
+  security::role_store& store,
+  security::role_name role_name,
+  security::role role) {
+    auto val_opt = store.get(role_name);
+    if (val_opt.has_value()) {
+        return val_opt.value() == role;
+    } else {
+        return false;
+    }
```

Comment:
```suggestion
    return store.get(role_name) == role;
```
But I think it's easier to avoid this function.

Reply:
Nice!